scanner/src/main/kotlin/storages/NoStorage.kt

```diff
@@ -37,5 +37,5 @@ class NoStorage : ScanResultsStorage() {
     override fun readFromStorage(pkg: Package, scannerDetails: ScannerDetails) =
         ScanResultContainer(pkg.id, emptyList())
 
-    override fun addToStorage(id: Identifier, scanResult: ScanResult) = false
+    override fun addToStorage(id: Identifier, scanResult: ScanResult) = AddResult(true)
```

Comment:
Because it is the intended behaviour of `NoStorage` to not store anything, reporting an issue in this case would be wrong.

Reply:
So, strictly speaking, this should have been `true` also in the original code? Meaning, again strictly speaking, there should have been a separate commit changing this from `false` to `true`, and then this commit only wrapping it in an `AddResult`, right? 😉

Reply:
Yes and no, I think the semantic slightly changed with this commit. I have added a sentence to the commit message:

```
    This slightly changes the semantics of the function, because prior the
    the return value described if the scan result was added to the storage
    and therefore the `NoStorage` implementation returned `false`. Now the
    return value describes if the operation was successful, therefore
    `NoStorage` now returns an `AddResult` with `success=true`, because not
    storing anything is the expected behavior of this implementation.
```

Reply:
At, thanks, I overlooked that :+1: